lib/ebanx.rb

```diff
@@ -85,13 +85,13 @@ def self.merge_default_params(command, params)
     params
   end
 
-  def self.request(command)
+  def self.request(command, as_json)
     uri = self::base_uri + command.request_action
 
     begin
       case command.request_method
       when :post
-        response = RestClient::Request.execute(:method => :post, content_type: command.response_type, :url => uri, :payload => command.params, :timeout => nil, :open_timeout => nil, :headers => self.format_user_data )
+        response = RestClient::Request.execute(:method => :post, content_type: command.response_type, :url => uri, :payload => as_json ? command.params.to_json : command.params, :timeout => nil, :open_timeout => nil, :headers => self.format_user_data )
```

Comment:
Could you extract the logic of the ternary into a method in order to improve code readability?

Reply:
Well... Ill extract into a variable. We're working with stateless code here, so if I would extract this into a method, iI would have to pass all the variables and the confusion would be the same.